### ISSUE TITLE
[Accessibility]: Profile/site-admin dropdown button has awkward pronounciation

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35314]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33515

### Problem description

The dropdown button for opening the siteadmin/profile dropdown speaks it as "Signed in as, clickable, at (@) clickable, <username> clickable.". 2.45 in the recording

### Expected behavior

Improve the pronouncation to be clearer (remove the @, no "clickable").

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by sourcegraph-bot-2** [comment]
Heads up @taylorsperry @jasongornall - the "team/frontend-platform" label was applied to this issue.

**Comment by umpox** [comment]
Related to https://github.com/sourcegraph/sourcegraph/issues/35267, closing this as it is more related to browser/screen reader behavior
